astacus/coordinator/plugins/zookeeper.py

```diff
@@ -117,6 +145,19 @@ def connect(self) -> ZooKeeperConnection:
         return KazooZooKeeperConnection(client)
 
 
+class KazooZooKeeperTransaction(ZooKeeperTransaction):
+    def __init__(self, request: TransactionRequest):
+        self.request = request
+
+    def create(self, path: str, value: bytes) -> None:
+        self.request.create(path, value)
+
+    async def commit(self) -> None:
+        results = await to_thread(self.request.client.retry, self.request.commit)
```

Comment:
Is this crucial piece of code tested somewhere?

Reply:
It's missing a test, I'll add it :+1:

Reply:
Well, good catch, there was a bug on that part :)